cirq-ionq/cirq_ionq/conftest.py

```diff
@@ -0,0 +1,5 @@
+import os
```

Comment:
We have added `conftest.py` for `cirq-ionq` and `cirq-google`. What about other modules like `cirq-core` and `cirq-aqt`? It's not needed there?

Reply:
cirq-core had it, but cirq-aqt didn't, good catch!